test/mock/frames/recursive.html

```diff
@@ -21,6 +21,6 @@
 </head>
 <body>
 	<h1>Recursive iframe</h1>
-	<iframe src="recursive.html">
+	<iframe src="nested1.html">
```

Comment:
`recursive.html` was infinitely including iframes. The infinite recursion was not required for that test to do what it was doing, so I changed it to a single iframe.